tests/vmi_lifecycle_test.go

```diff
@@ -1628,6 +1628,37 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 		})
 	})
 
+	Describe("Reset a VirtualMachineInstance", func() {
+		const vmiLaunchTimeout = 360
+
+		It("should succeed", func() {
+			vmi := libvmops.RunVMIAndExpectLaunch(libvmifact.NewAlpineWithTestTooling(), vmiLaunchTimeout)
+
+			By("Checking that the VirtualMachineInstance console has expected output")
+			Expect(console.LoginToAlpine(vmi)).To(Succeed())
+
+			errChan := make(chan error)
+			go func() {
+				time.Sleep(5)
+				errChan <- kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Reset(context.Background(), vmi.Name)
+			}()
+
+			start := time.Now().UTC().Unix()
+			err := waitForVMIReset(vmi)
+			end := time.Now().UTC().Unix()
+
+			if err != nil {
+				err = fmt.Errorf("start [%d] end [%d] err: %v", int32(start), int32(end), err)
+			}
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
Do we really need these time stamps?

```suggestion
			Expect(waitForVMIReset(vmi)).To(Succeed())
```

Reply:
is there a problem with the timestamps? they are informational